Makefile

```diff
@@ -258,3 +266,24 @@ kind-load-images:
             $(REGISTRY)/test-backend:dev \
             $(REGISTRY)/jmalloc/echo-server:dev \
             $(REGISTRY)/busybox:dev
+
+
+GATEWAY_API_PACKAGE ?= sigs.k8s.io/gateway-api
+GATEWAY_API_VERSION ?= v0.5.1
+GATEWAY_API_CRDS_LOCAL_PATH = $(shell go env GOPATH)/pkg/mod/$(GATEWAY_API_PACKAGE)@$(GATEWAY_API_VERSION)/config/crd
```

Comment:
If we already keep a copy of this CRD configuration in the our repo, why not just use this copy to install?

Reply:
Dependent updates are convenient. The CRDs in the repo as an example, and maybe we can delete it?